en/docs/getting-started/basic-architecture.md

```diff
@@ -0,0 +1,78 @@
+ 
+## Control Plane
+ 
+ Control plane is where the API security validation, API key generation and rate limiting decisions are taken.
+ 
+### Components
+ 
+#### Key Manager
+ 
+ ![Key Manager]({{base_path}}/assets/img/get_started/key-manager-overview.png)
+
+ The Key Manager is the identity provider for the WSO2 API Manager and acts as the Secure Token Service(STS). 
+ 
+ In WSO2 API Manager, tokens are generated for an application. The Key manager provides a token API to generate access tokens. These tokens can be used by clients to invoke APIs exposed by WSO2 API Manager. Furthermore, 
+ The Key Manager exposes a revoke API to revoke any access token. The Key Manager supports OAuth2.0, Basic Auth, Mutual SSL as well as API-Key based authentication mechanisms. 
+ 
+ WSO2 API Manager only supports creating applications that support JWT formatted OAuth2.0 type tokens.
+ A client can generate a JWT formatted OAuth 2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, an API Key can be generated through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token.
+ 
+ When a client invokes an API with a JWT formatted OAuth2.0 access token or an API Key,  the Gateway validates the token by validating its signature and subscription.
+ 
+ WSO2 API Manager 3.2.0 can accommodate a third-party Key Manager in place of its Key Manager component and also accommodate multiple Key Managers such as Okta, OAuth0, KeyCloak, etc. 
+ 
+ 
+#### Traffic Manager
+ 
+  ![Traffic Manager]({{base_path}}/assets/img/get_started/traffic-manager-overview.png)
+ 
+The Traffic Manager helps users to regulate API traffic, make APIs and applications available to consumers at different service levels, and secure APIs against security attacks. The Traffic Manager features a dynamic throttling engine to process throttling policies in real-time, including rate-limiting of API requests. 
+
+In addition to the above, the traffic manager also helps to keep the API Gateway's in-memory map, used for key validation, up-to-date. The traffic manager updates the databases based on artifact(API/application) update events received from the API Publisher and API Developer Portal.  The Traffic Manager then goes on to notify the API Gateway and help it update its in-memory map via REST APIs.
+
```

Comment:
I think we need to re-word this paragraph. Couple of this to point out
1- traffic manager does not update the database. it receives events from publisher/devportal on changes such as API create, app create etc and put them in to a jms topic. Gateways which are subscribed to this topic get this message to them and it updates it's inmemory maps based on the message.